### ISSUE TITLE
Truncating newest event on homepage

### ISSUE THREAD

**Issue by Wilto** [bostonjs/bostonjs.com#22]
We’re showing a lot of copy on the homepage, and offering a way to click through to the same full text—we might want to consider a way of highlighting some promotional copy here (or truncated, worst case), and offer the full text behind a cut.

I feel like this may have come up in discussion before; if so, I’m just filing an issue so I’m not relying on feeling like things have come up in discussion before.

**Comment by ajpiano** [comment]
Yeah, one thing to figure out is how to do this since all the data is coming from meetup and needs to be functional there too. Hrmzies.
